components/molecule/collapsible/src/index.js

```diff
@@ -90,6 +95,10 @@ const MoleculeCollapsible = ({
 MoleculeCollapsible.displayName = 'MoleculeCollapsible'
 
 MoleculeCollapsible.propTypes = {
+  /**
+   * Container align center || right
+   */
+  alignContainer: PropTypes.oneOf(Object.values(CONTENT_ALIGN)),
```

Comment:
I don't see the need to do it, we already have a Sass variable that allows us to customise it from our sites.

Reply:
I'd rather use a finite number of JS props to define components customizable properties instead of modifying Sass values. Feels more declarative to me.

Reply:
Yep, I was wrong, in this case we could want more than one collapsible in our sites aligned differently, so it makes a lot of sense 👍